README.md

```diff
@@ -145,20 +145,27 @@ This plugin creates device plugin endpoints based on the configurations given in
 {
     "resourceList": [{
             "resourceName": "intel_sriov_netdevice",
-            "selectors": {
+            "selectors": [{
                 "vendors": ["8086"],
                 "devices": ["154c", "10ed"],
                 "drivers": ["i40evf", "ixgbevf"]
-            }
+            }]
         },
         {
             "resourceName": "intel_sriov_dpdk",
-            "selectors": {
-                "vendors": ["8086"],
-                "devices": ["154c", "10ed"],
-                "drivers": ["vfio-pci"]
-            }
-        }
+            "selectors": [
+                {
+                    "vendors": ["8086"],
+                    "devices": ["154c", "10ed"],
+                    "drivers": ["vfio-pci"]
+                },
+                {
+                    "vendors": ["15b3"],
+                    "devices": ["1014"],
+                    "drivers": ["mlx5_core"]
```

Comment:
nit. Although this won't cause any issue with above configuration (using ``mlx5_core`` under ``intel_sriov_dpdk`` resourcename), it might be worth changing it to intel kernel driver like ``i40evf``;  and we could add ``mlx5_core`` driver in the first example with mlx_sriov as the resource name:
```
{
    "resourceList": [{
	            "resourceName": "mlx_sriov",
	            "selectors": [{
	                "vendors": ["15b3"],
	                "devices": ["1014"],
	                "drivers": ["mlx5_core"]
	            }]
	        },
                {
	            "resourceName": "intel_sriov",
                    "selectors": [
	                {
	                    "vendors": ["8086"],
	                    "devices": ["154c", "10ed"],
	                    "drivers": ["vfio-pci"]
	                },
	                {
	                    "vendors": ["8086"],
	                    "devices": ["154c"],
	                    "drivers": ["i40evf"]
                       }
	            ]
                }
	    ]
}
...

```

Reply:
fixed